arcade/scene.py

```diff
@@ -20,17 +21,18 @@ class Scene:
     https://api.arcade.academy/en/latest/tutorials/views/index.html
 
     Attributes:
-        :sprite_lists: A list of `SpriteList` objects. The order of this list is the
+        :DON'T CHANGE __sprite_lists or __name_mapping
+        :__sprite_lists: A list of `SpriteList` objects. The order of this list is the
                        order in which they will be drawn.
-        :name_mapping: A dictionary of `SpriteList` objects. This contains the same lists
-                       as the `sprite_lists` attribute, but is a mapping of them by name. This is
-                       not necessarily in the same order as the `sprite_lists` attribute.
+        :__name_mapping: A dictionary of `SpriteList` objects. This contains the same lists
+                       as the `__sprite_lists` attribute, but is a mapping of them by name. This is
+                       not necessarily in the same order as the `__sprite_lists` attribute.(Dictionaries have no order)
```

Comment:
If these are non-public this entire section should just be removed.

Reply:
that's true

Reply:
Fixed